### PR TITLE
[zh] Sync Configuring istioctl for a remote cluster blog into Chinese

### PR DESCRIPTION
**Pull request by wilsonwu**
Translate Configuring istioctl for a remote cluster blog into Chinese:
```
content\zh\blog\2022\istioctl-proxy\index.md
```

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/blog/2022/istioctl-proxy/index.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
+---
+title: 为远程集群配置 istioctl
```
Comment (windsonsea) [inline content/zh/blog/2022/istioctl-proxy/index.md:2]:
```suggestion
title: 为从集群配置 istioctl
```

**Review by windsonsea — COMMENTED** [review]
理论上，remote cluster 这种 term，需要跟 [glossary/remote-cluster.md](https://github.com/istio/istio.io/blob/master/content/zh/docs/reference/glossary/remote-cluster.md) 保持一致。

当然有一多半页面直译成了`远程集群` -_!

**Comment by wilsonwu** [comment]
> 理论上，remote cluster 这种 term，需要跟 [glossary/remote-cluster.md](https://github.com/istio/istio.io/blob/master/content/zh/docs/reference/glossary/remote-cluster.md) 保持一致。
> 
> 当然有一多半页面直译成了`远程集群`，我们的文档一致性、对 glossary 的遵从度还远远不够 -_!

好的，那么我都改成从集群吧，跟随规范，从我做起。

File: content/zh/blog/2022/istioctl-proxy/index.md (hunk 1 above)
Comment (wilsonwu) [reply content/zh/blog/2022/istioctl-proxy/index.md]: Fixed in the latest commit.

**Review by windsonsea — APPROVED** [review]
LGTM

**Pull request merged**